### PR TITLE
Feat: Adding grouping direction hidden method for the table grouping

### PR DESCRIPTION
**Pull request by bensondevs**
## Description

I have a client that asks me to disable the sorting direction of the table group because we want to use the specified sorting direction as preset configuration that cannot be changed. To enable that for the user, we need to add the hide feature for the grouping direction for the user.

## Visual changes

Before:
image
image

After:
image
image

## Functional changes

- [ x ] Code style has been fixed by running the `composer cs` command.
- [ x ] Changes have been tested to not break existing functionality.
- [ x ] Documentation is up-to-date.

### CODE REVIEW

File: packages/tables/resources/views/index.blade.php [hunk 1]
```diff
@@ -42,6 +42,7 @@
     $isReorderable = $isReorderable();
     $isReordering = $isReordering();
     $areGroupingSettingsVisible = (! $isReordering) && count($groups) && (! $areGroupingSettingsHidden());
+    $isGroupingDirectionHidden = (! $isGroupingDirectionHidden());
```
Comment (danharrin) [inline packages/tables/resources/views/index.blade.php]: This doesn't make logical sense with the !, the value is true if its actually false

Comment (danharrin) [reply packages/tables/resources/views/index.blade.php]: You'll need to swap around the `@if` statements in the groups.blade.php view to fix the change as well

**Review by danharrin — CHANGES_REQUESTED** [review]

File: packages/tables/resources/views/index.blade.php (hunk 1 above)
Comment (danharrin) [reply packages/tables/resources/views/index.blade.php]: Or, a better alternative could be renaming to `$isGroupingDirectionVisible` and passing the groups.blade.php attribute as just `:direction="true / false"` which I think is still clear?

Comment (bensondevs) [reply packages/tables/resources/views/index.blade.php]: Updated as suggested, please check

Comment (danharrin) [reply packages/tables/resources/views/index.blade.php]: Thanks

**Pull request merged**